content/es/registrar/la.md

```diff
@@ -12,6 +12,7 @@ ip_deadline = "hasta 30 días antes del día de las elecciones"
 online_deadline = "hasta 20 días antes del día de las elecciones"
 mail_received_deadline = ""
 mail_postmarked_deadline = "debe tener el sello del correo 30 días antes del día de las elecciones"
+dates_electionday = "শনিবার, ১০ ডিসেম্বর, ২০২২"
```

Comment:
This looks like it was an error. Spanish data files should not be modified as part of this branch.

Reply:
Fixed